.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 yarn.lock
 dist/
+.vscode/
```

Comment:
This should be in your own global gitignore, not here.

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c